controllers/registerController.js

```diff
@@ -1,6 +1,24 @@
 // import database
 const db = require('../db');
+const nodemailer = require('nodemailer');
+const sendgridTransport = require('nodemailer-sendgrid-transport')
 
+const config = require('config');
+const sgapiKey = config.get('api_key.SENDGRID');
+// create reusable transporter object using sendgridTransport, which
+// returns a configuration that nodemailer can use, to use sendgrid
+const transporter = nodemailer.createTransport(sendgridTransport({
+    auth: {
+        api_key: sgapiKey
+    }
+}));
+
+let message = {
+    to: 'aiaimforworld@gmail.com, benevbright@gmail.com',
+    from: 'admin@catsdogssoftball.com',
+    subject: 'We got a registraion to our team!',
+    html: '<h2> We got a registraion to our team!</h2>'
```

Comment:
I would love to get the new player's email address in the email body.

Reply:
I'd love to, too, and have tried but I couldn't. need help.:)